static-build/test/static-build/exports.test.lua

```diff
@@ -127,10 +127,48 @@ local check_symbols = {
     'HMAC_Update',
     'HMAC_Final',
 
+
```

Comment:
Please, don't forget to add a prefix for a commit title: 'test' in the case.

Reply:
Done